packages/api-explorer/src/ResponseExample.jsx

```diff
@@ -215,6 +213,29 @@ class ResponseExample extends React.Component {
                 let example;
                 const mediaTypes = ex.languages;
 
+                // If this response has no documented languages/media types we should render out its status message as
+                // the documented example instead of showing an empty box.
+                if (!mediaTypes.length) {
+                  let statusMessage;
+                  try {
+                    const code = getStatusCode(ex.status);
+                    statusMessage = code.message;
+                  } catch (err) {
+                    statusMessage = `HTTP Status Code: ${ex.status}`;
+                  }
+
+                  return (
+                    <div key={index}>
```

Comment:
i know we're doing this below as well, but if there's anything more unique that we have than the index that'd be better

Reply:
Modified it to `response-{index}`